requirements.txt

```diff
@@ -0,0 +1,73 @@
+aniso8601==4.0.1
```

Comment:
The `requirements.txt` should be empty as per now as you should avoid having dependencies that are not being used.

Reply:
Noted